src/OrchardCore.Modules/OrchardCore.Search.Lucene/Permissions.cs

```diff
@@ -4,12 +4,14 @@ namespace OrchardCore.Search.Lucene;
 
 public sealed class Permissions : IPermissionProvider
 {
+    private readonly LuceneIndexSettingsService _luceneIndexSettingsService;
+
+    [Obsolete("This will be removed in a future release. Instead use 'LuceneSearchPermissions.ManageLuceneIndexes'.")]
```

Comment:
all there should be pointing to the new properties instead of creating new objects